ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -189,7 +199,9 @@ const IgvPanel = React.memo((
     size,
   )
 
-  const tracks = rnaReferences.concat(getIgvTracks(igvSampleIndividuals, individualsByGuid, sampleTypes))
+  const rnaRefs = updateJunctionRefs(rnaReferences, minJunctionEndsVisible)
```

Comment:
this seems odd to me. Do we actually want to update the reference track display as well?

Reply:
So in that case, I think it is a bit cleaner to post-process all the track together, the way TGG viewer does, rather than separately updating the reference tracks and regular tracks. Also, l I think the name `updateJunctionRefs` is not as descriptive as it could be and currently seems to only handle merged tracks and not pure junction tracks. I would name it something more generic in case we add any other settings that require post processing in the future, maybe something like `applyUserTrackSettings`. So the change here would just be
```
const tracks = applyUserTrackSettings(rnaReferences.concat(getIgvTracks(igvSampleIndividuals, individualsByGuid, sampleTypes)))
```